src/snowcli/cli/render/commands.py

```diff
@@ -142,16 +142,35 @@ def render_template(
             key, value = _parse_key_value(key_value_str)
             data[key] = value
 
+    generic_render_template(
+        template_path=template_path, data=data, output_file_path=output_file_path
+    )
+
+
+def generic_render_template(
+    template_path: Path, data: dict, output_file_path: Optional[Path]
```

Comment:
```suggestion
    template_path: Path, data: dict, output_file_path: Optional[Path] = None
```
Because the file is optional